### PR TITLE
August 17 learn team meeting notes

### PR DESCRIPTION
**Pull request by toddjordan**
Feel free to make corrections if you were there.

Thumbsup this comment to approve.

### CODE REVIEW

File: docs-team/2017-08/august-17.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+[ ] DB: pending Leah creating Heroku org for Ember to shift to new org approach
+[ ] DB: Heroku blog post
+[ ] RT: Write Rolodex/Contact Book Tutorial to launch with 2.16
+[ ] TJ/RM: record/task out stuff that needs to be done for the api changes in 2.16
+
+## Minutes
+
+### Attendees (Initials: Name @slack-id)
+- TJ: Todd Jordan @todd.jordan
+- RM: Richardo Mendes
```
Comment (jkarsrud) [inline docs-team/2017-08/august-17.md]: RM: Ricardo Mendes (plus slack-id)

File: docs-team/2017-08/august-17.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+[x] MS: Will finish route restructuring @Michael Sk
+[x] RM: Create an issue for adding heroku logo to site
+[ ] YB: clean up the tutorial PR to the website
+[ ] RM, KG: Pair on proposed Guides ToC  (after ember camp)
+[x] TJ: Go/No Go call on API Doc (Monday)
+[ ] RT: update on discourse+blog integration
+[ ] DB: pending Leah creating Heroku org for Ember to shift to new org approach
+[ ] DB: Heroku blog post
+[ ] RT: Write Rolodex/Contact Book Tutorial to launch with 2.16
+[ ] TJ/RM: record/task out stuff that needs to be done for the api changes in 2.16
```
Comment (jkarsrud) [inline docs-team/2017-08/august-17.md]: Add dashes in front of `[ ]` to create a proper task list

File: docs-team/2017-08/august-17.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+DB: sounds like a nice to have and not for 2.16, plus not sure that its something we want to add to middleman if we are going to kill it.
+
+RM: we could add it as a glimmer component.
+
+TJ: so yeah we need to continue to go to the right place.  As long as ember supports both ways of import we should
+
+DB: but we would need to do that forever because our docs go back to 1.x
+
+RM: we could cut the app into do different things at a specific version.
```
Comment (jkarsrud) [inline docs-team/2017-08/august-17.md]: "(...)cut the app *to do* different things(...)"?

File: docs-team/2017-08/august-17.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+We should probably consider 2.16 as a different app, because the mental model/organization is significantly different.  not sure how that translates into urls/etc, but the current viewer needs changes.  (privates, toc, etc).
+
+TJ: we could implement that in a couple of different ways.  We could just keep one app and then have one set of routes go to the current view (today), and have the new routes go to 2.16+
+RM: we should figure out if we truly are having different urls.
```
Comment (jkarsrud) [inline docs-team/2017-08/august-17.md]: Add another line break, I read this as "2.16+ RM" in the rendered version

File: docs-team/2017-08/august-17.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+
+#### Algolia
+
+MS: I got back into it and pushed to pull requests, 1 to update selectors to help algolia scrape, the other to re-add the search.  looking into an issue, but then we will be ready to go.  it is deployed as a heroku preview if folks want to check it out.   Should take long to be ready.
+https://ember-api-docs-staging-pr-290.herokuapp.com/api/ember/2.14/namespaces/Ember
```
Comment (jkarsrud) [inline docs-team/2017-08/august-17.md]: Should or shouldn't take long?

**Comment by toddjordan** [comment]
Thanks @jkarsrud ❤️

File: docs-team/2017-08/august-17.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+
+#### new sections
+
+RM: 2 big sections that I want to add is SSR (fastboot) and a deployment section.  Deployment not just ember-cli-deploy, (though we would mention) but rather general deployment options
+
+#### API docs
+
+RM: Yehuda made a spreadsheet to categorize major sections.  Ties into restructuring of the guides.  Goal it to make ember consumable for new users.
+
+https://docs.google.com/spreadsheets/d/1UNfIKGIigWnabCUV6aQK-zBTuv8LVBA8Kzkk1bVhZjQ/edit
```
Comment (acorncom) [inline docs-team/2017-08/august-17.md]: Do we want this public record?

Comment (locks) [reply docs-team/2017-08/august-17.md]: Seems fine!

**Pull request merged**